resourcemgr/resourcemgr.c

```diff
@@ -1932,6 +1933,12 @@ TSS2_RC ResourceMgrReceiveTpmResponse(
                                 foundEntryPtr->hierarchy, cmdConnectionId );
                         if( rval != TSS2_RC_SUCCESS )
                             goto returnFromResourceMgrReceiveTpmResponse;
+
+                        // Find entry for the new persistent object and flag it as persistent
+                        rval = FindEntry( entryList, RMFIND_REAL_HANDLE, persistentHandle, &foundEntryPtr );
+                        if( rval != TSS2_RC_SUCCESS )
+                            goto returnFromResourceMgrReceiveTpmResponse;
+                        foundEntryPtr->status.persistent = 1;
```

Comment:
ok. Actually I tried the way to looking at the handle type before I tried to using persistent bit. I choosed the persistent bit way since it make my change looks smaller and easy to understand. I can make new PR to remove the persistant bit.